### PR TITLE
[WIP] initial linux support

### PR DESCRIPTION
**Pull request by corpix**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,10 @@
 language: node_js
 
+# XXX: because it has 7.5GB of memory
+# instead of 4GB in container
+# see: https://docs.travis-ci.com/user/reference/overview/
+sudo: true
```
Comment (corpix) [inline .travis.yml]: https://pbs.twimg.com/media/B7GP8UjCUAETNWs.jpg

Comment (olegakbarov) [reply .travis.yml]: YES =(

**Review by olegakbarov — APPROVED** [review]
LGTM

**Comment by corpix** [comment]
Superseded by https://github.com/cryptounicorns/desktop/pull/5

**Pull request closed without merging**